projects/npm-tools/packages/npm-scripts/test/utils/parseBnd.js

```diff
@@ -3,25 +3,14 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-const path = require('path');
-
 const parseBnd = require('../../src/utils/parseBnd');
+const getFixturePath = require('../../support/getFixturePath');
 
 describe('parseBnd()', () => {
-	function getBndPath() {
-		return path.join(
-			__dirname,
-			'..',
-			'..',
-			'__fixtures__',
-			'utils',
-			'parseBnd',
-			'bnd.bnd'
-		);
-	}
+	const bndPath = getFixturePath('utils/parseBnd/bnd.bnd');
 
-	it('correctly parses a bnd.bnd file', () => {
-		const entries = parseBnd(getBndPath());
+	it('correctly parses a bnd.bnd file', async () => {
```

Comment:
Why `async`? I can't see any `await` in here.

Reply:
Oops, that's a leftover because I first tried with `getFixture`. 
I thinks there's an eslint rule for that :thinking: ...